rlp/unsafe.go

```diff
@@ -14,8 +14,8 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
-//go:build !nacl && !js && cgo
-// +build !nacl,!js,cgo
+//go:build !nacl && !js && !wasip1 && cgo
+// +build !nacl,!js,!wasip1,cgo
```

Comment:
WASM supports pointers, so this file should be OK to just leave the tags as-is.

Reply:
it does, I didn't look too closely at why things were tagged.  Thanks for pointing it out, I'll revert the file.